modules/uhttp/filters_test.go

```diff
@@ -66,9 +73,11 @@ func TestTracingFilterWithLogs(t *testing.T) {
 		assert.True(t, len(buf.Lines()) > 0)
 		var spanFinished bool
 		for _, line := range buf.Lines() {
-			assert.Contains(t, line, "traceID")
-			assert.Contains(t, line, "spanID")
-			spanFinished = spanFinished || strings.Contains(line, "Span finished")
+			if strings.Contains(line, "Span finished") {
```

Comment:
I enabled logging from the jaeger library for this test. This logger is assigned on creating the tracer which is at service startup and hence will not have traceID or spanID. Essentially, we now have lines logged by the jaeger library.